ci/audit.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-nancy
```

Comment:
It may be better to add Nancy to the DP-concourse-tools-node-go Docker image so that we can still do npm audit when we get to that story.